versions.json

```diff
@@ -1,34 +1,38 @@
 [
   {
-    "version": "6.8-rc4",
+    "version": "6.9-rc3",
     "type": "mainline"
   },
   {
-    "version": "6.7.4",
+    "version": "6.8.4",
     "type": "stable"
   },
   {
-    "version": "6.6.16",
+    "version": "6.7.12 ",
+    "type": "stable"
```

Comment:
I think the github action is going to trip over that since it'll try to tag 6.7 and 6.8 as stable. Would you mind adjusting the script to filter out `[EOL]`?

Reply:
Sure, will do. It's a bit of a mess figuring this out using XPath. Given that the releases are also published in JSON format at https://www.kernel.org/releases.json which will make filtering a lot easier, would you mind if I change the update script to use that file as a source?

Reply:
@lmb implemented using https://www.kernel.org/releases.json and `jq` JSON transformation, PTAL.